kafka-rest/src/main/java/io/confluent/kafkarest/v2/KafkaConsumerManager.java

```diff
@@ -367,19 +367,16 @@ protected <T> RunnableFuture<T> newTaskFor(Runnable runnable, T value) {
 
   class RunnableReadTask implements Runnable, Delayed {
     private final ReadTaskState taskState;
-    private final Instant started;
     private final Instant requestExpiration;
     private final Duration backoff;
     // Expiration if this task is waiting, considering both the expiration of the whole task and
     // a single backoff, if one is in progress
     private Instant waitExpiration;
-    private final Clock clock = Clock.systemUTC();
```

Comment:
we already have a static clock on top-level class, there is no need to have another clock here.